librz/core/cconfig.c

```diff
@@ -1737,6 +1737,180 @@ RZ_API bool rz_core_esil_cmd(RzAnalysisEsil *esil, const char *cmd, ut64 a1, ut6
 	return false;
 }
 
+static void eval_string(RzConfig *cfg, char *name) {
+	if (!*name) {
+		return;
+	}
+	char *eq = strchr(name, '=');
+	if (eq) {
+		*eq++ = 0;
+		rz_str_trim(name);
+		rz_str_trim(eq);
+		if (*name) {
+			(void)rz_config_set(cfg, name, eq);
+		}
+	} else {
+		if (rz_str_endswith(name, ".")) {
+			RzCmdStateOutput state = { 0 };
+			rz_cmd_state_output_init(&state, RZ_OUTPUT_MODE_QUIET);
+			rz_core_config_print_all(cfg, NULL, &state);
+			rz_cmd_state_output_print(&state);
+			rz_cmd_state_output_fini(&state);
+		} else {
+			const char *v = rz_config_get(cfg, name);
+			if (v) {
+				rz_cons_printf("%s\n", v);
+			} else {
+				rz_cons_printf("Invalid config key %s\n", name);
+			}
+		}
+	}
+}
+
+RZ_API bool rz_core_config_eval_and_print(RzCore *core, const char *str, bool many) {
```

Comment:
`many` is not required anymore. it was used before, but in newshell it's not needed. you can remove the argument and the path depending on it.

Reply:
Actually...  I'm starting to doubt the usefulness of the whole function. @XVilka it seems to be used only in rizin.c.. Do you see anything useful that can be done there that cannot be done with `rz_config_get`+`rz_cons_printf`?

Reply:
I think we can indeed remove it completely it seems.

Reply:
Looks like this broke a gazillion of tests. Should we make this a part of another PR or something?

Reply:
@officialcjunior the breakage looks like because `-escr.color=0` didn't work in `binrz/rz-test/run.c`. Just inserting space should fix the problem.